app/assets/src/components/ui/controls/dropdowns/BareDropdown.jsx

```diff
@@ -1,6 +1,13 @@
 // This component allows you to easily add a dropdown of options to any trigger element
 // Wraps around Semantic UI Dropdown, with react-popper as an alternative.
 
+// NOTE ABOUT CUSTOMIZING THE DROPDOWN:
+// If you don't want the user to be able to select a value, and you want to specify
```

Comment:
it sounds a little confusing to say "you don't want the user to be able to select a value"? you just mean not select within this component right?

Reply:
Yeah, for example in the User Menu, the user isn't selecting a value, they're picking from a list of navigation. Will think about how to word this. Thanks for feedback.

Reply:
A good way to structure the use cases is a bullet list with a key title. Something like: 
```
* options with default rendering that return a value: ...
* options with custom rendering that return a value: ...
* options that trigger an immediate action (e.g. navigations links): ...
```